README.md

```diff
@@ -110,7 +116,7 @@ ALLOW_INSECURE_SSL=true python myscript.py
 WARNING! You should not need to set this when using the SDK for cloud zegami
 
 # In Development
-This SDK is in active development, not all features are available yet. Creating/uploading to collections is not supported currently - check back soon!
+This SDK is in active development. Features are actively being developed according to user feedback. Please share your suggestions or fork this repository feel free to raise a PR
```

Comment:
Please share your suggestions or fork this repository **and** feel free to raise a PR.